src/clap_app.rs

```diff
@@ -329,6 +329,24 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                 .takes_value(true)
                 .value_name("width")
                 .hidden_short_help(true)
+                .validator(
+                    |t| {
+                        let sign = t.chars().next().unwrap();
+                        let num = if (sign == '+') || (sign == '-') {
```

Comment:
Instead if constructing `num`, we could also try to `.parse` into a `i32` and then fail if the result is zero and `is_offset` is false.

Reply:
Part of the validation I implemented is checking that `z123` isn't considered a valid terminal width. Would `.parse::<i32>()` be able to handle a positive sign character?

If so, that should be able to work.